java-package/native/src/main/resources/META-INF/backport.txt

```diff
@@ -0,0 +1,2 @@
+The file is based off Apache MXNet 1.7 branch (https://github.com/apache/incubator-mxnet/commit/7a84fe11dd7399d1c2bdd0fb679c46af69e48861)
```

Comment:
check what does this line do

Reply:
This file gets removed because it has never been used in this project.